### PR TITLE
fix(ci): update Cypress to 6.4.0

### PR DESCRIPTION
**Pull request by tsusanka**
This was not the culprit at the end but since I did it we can just merge it. Should not break anything :tm:.

cc @mroz22 if you could take a look.

### CODE REVIEW

**Review by slowbackspace — APPROVED** [review]

File: ci/packages/suite-web.yml [hunk 1]
```diff
@@ -77,7 +77,7 @@ suite-web deploy dev:
     COMPOSE_FILE: ./docker/docker-compose.suite-ci.yml
     # corresponds with the NextJs assetPrefix. Tells Cypress where is the index of application
     CYPRESS_ASSET_PREFIX: /web
-    CYPRESS_baseUrl: ${DEV_SERVER_URL}/suite-web/${CI_BUILD_REF_NAME}
+    CYPRESS_baseUrl: ${DEV_SERVER_URL}/suite-web/${CI_BUILD_REF_NAME}/
```
Comment (mroz22) [inline ci/packages/suite-web.yml]: doesn't this break anything? I would expect it to break asset prefix. After this change cypress might see something like 'sldev.cz/branch//web'

Comment (tsusanka) [reply ci/packages/suite-web.yml]: Ok I rather remove it.

Comment (tsusanka) [reply ci/packages/suite-web.yml]: Done. I have force-pushed directly, hopefully you'll forgive me.

**Comment by mroz22** [comment]
ok, it works! the test suite failed only because of one flaky test :(

**Pull request merged**